src/handlers/guild.ts

```diff
@@ -622,3 +629,148 @@ export function getGuild(guildID: string, counts = true) {
     { with_counts: counts },
   ) as Promise<UpdateGuildPayload>;
 }
+
+/** Returns the guild template if it exists */
+export function getGuildTemplate(
+  guildID: string,
+  templateCode: string,
+): Promise<Template> {
+  return RequestManager.get(
+    `${endpoints.GUILD_TEMPLATES(guildID)}/${templateCode}`,
+  ) as any;
+}
+
+/**
+ * Create a new guild based on a template
+ * NOTE: This endpoint can be used only by bots in less than 10 guilds.
+ */
+export async function createGuildFromTemplate(
+  templateCode: string,
+  data: CreateGuildFromTemplate,
+) {
+  if (cache.guilds.size >= 10) {
```

Comment:
this should use cachehandlers

Reply:
@Skillz4Killz, can you provide an example?

Reply:
1. Add a .size option here https://github.com/Skillz4Killz/Discordeno/blob/0c8731679c376c2b75f3df3ca6401623fbce1912/src/controllers/cache.ts#L95
2. if (await cacheHandlers.size("guilds") >= 10) {

}

Reply:
But that is irrelevant to the PR, I don't think that should be done in this PR.

Reply:
#188